dashboard/app/controllers/api/v1/amazon_future_engineer_controller.rb

```diff
@@ -31,7 +31,7 @@ def submit
     )
   rescue Services::AFEEnrollment::Error => e
     Honeybadger.notify e
-    head :bad_request
+    render json: e.to_s, status: 400
```

Comment:
Still respond with BAD REQUEST, but also include the error text in the body of the response, which may help with support later.